brave/brave-core
# Storybook: Created Portfolio Tab View

Resolves <https://github.com/brave/brave-browser/issues/15123>

Created the Portfolio Tab View and added it too storybook under Concepts.
Also needed to add 2 new dependancies from "Recharts" to create the line chart component.

Package: https://www.npmjs.com/package/recharts
Typescript Package: https://www.npmjs.com/package/@types/recharts

Will need to run `npm install`

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. `npm run storybook`
2. Confirm that storybook shows the new component.
3. Check functionality of the new component.

https://user-images.githubusercontent.com/40611140/113927389-a695c600-97aa-11eb-8596-b6be23706e5c.mov
